### ISSUE TITLE
"convert to ES6 module" breaks code formatting

### ISSUE THREAD

**Issue by jamie-pate** [Microsoft/TypeScript#24693]
**TypeScript Version:**  2.90-insiders.20180503 (fixed in 3.0.0-dev.20180605)

**Search Terms:** 
"convert to ES6 module" formatting fixes else newline "newline before else"

**Code**
```
-        Object.keys(properties).forEach(function(key) {
+        Object.keys(properties).forEach(function (key) {
             var value = properties[key];
             if (value && typeof value.valueOf() === 'object') {
                 Object.assign(element[key], value);
-            } else {
+            }
+            else {
                 element[key] = value;
             }
         });
     }

```

**Expected behavior:**
code style for `} else {` should be preserved.

**Actual behavior:**
a newline is added after `}` and there is no setting to prevent this behavior.

**Related Issues:**
https://github.com/Microsoft/vscode/issues/19335
https://github.com/Microsoft/vscode/issues/21723

**Comment by jamie-pate** [comment]
Looks like it's fixed in 3.0.0-dev.20180605

**Comment by jamie-pate** [comment]
fixed in 3.0.0-dev.20180605, just thought i'd leave documentation since I couldn't find anything
